docs/python/tutorial-django.md

```diff
@@ -233,13 +234,16 @@ Debugging gives you the opportunity to pause a running program on a particular l
 1. Replace the contents of `views.py` with the following code to define the `hello_there` function that you can step through in the debugger:
 
     ```python
-    from django.http import HttpResponse
-    from datetime import datetime
     import re
+    from datetime import datetime
+
```

Comment:
I really prefer to not have all these extra blank lines in the code, regardless of what a formatter want to impose. For the purposes of inline code in a tutorial, it's much better to keep the code tight.